tests/Integration/AuthOnboardingFirstLoginTest.php

```
@@ -0,0 +1,51 @@
+                && (int) $stored['id'] === $userId
+                && (int) $stored['org_id'] === $orgId
+        ));
+
+        $auth = new Auth($session, $this->db);
+        $auth->login($user);
+
+        $row = $this->db->query("SELECT onboarding_seen_at FROM users WHERE id = ?", [$userId])->fetch();
+
+        self::assertNotEmpty($row['onboarding_seen_at'] ?? null);
+        self::assertTrue((bool) ($_SESSION['_show_onboarding_wizard'] ?? false));
+    }
```

PHPMD (2.15.0)

[error] firstLoginPersistsOnboardingSeenAndSetsOneTimeSessionFlag accesses the super-global variable $_SESSION. (undefined)

(Superglobals)